### ISSUE TITLE
aarch64-linux support?

### ISSUE THREAD

**Issue by L-as** [Admicos/minecraft-wayland#32]
I'm trying to get Minecraft working on aarch64-linux, and this is obviously part of that.
I'm using the Freedreno VK backend along with Zink, on Minecraft 1.19.2 on NixOS with Sway (albeit this is probably not relevant) with the GLFW revision and patches specified in bdc3c0d192097459eb4e72b26c8267f82266e951.

[crash-2022-08-29_16.49.31-client.txt](https://github.com/Admicos/minecraft-wayland/files/9446437/crash-2022-08-29_16.49.31-client.txt)

The error message is very unhelpful, and I'm wondering if anyone has any idea what it could mean.
It seems like I'll have to add some printfs to GLFW.

Is there perhaps some way to get Minecraft to emit more information, or?

Thank you.

**Comment by L-as** [comment]
This was very stupid.... I was just missing libwayland-client lol. I got it working. Nothing extraordinary was necessary, albeit I made my own simpler fork of GLFW: https://github.com/L-as/glfw/commit/846e2b4d881523803afcb393e624c363eef6d4b8

You're free to use this instead (obviously, given the license), and it is IMO a cleaner solution since it catches all the issues stemming from Minecraft not correctly handling GLFW errors.
I use Sway so I didn't need any libdecoration stuff.
